### ISSUE TITLE
error while importing package : `Unexpected token import`

### ISSUE THREAD

**Issue by kansi** [rubenspgcavalcante/leaflet-ant-path#12]
Details:
- leaflet version 0.7.7 installed via npm
- leaflet-ant-path version 0.1.3 installed via npm

error occurs at the following line 
https://github.com/rubenspgcavalcante/leaflet-ant-path/blob/master/src/plugin/main.js#L1

if my understanding is right then the issue is the leaflet-ant-path is not getting transpiled to javascript because I can see the es6 code in my js bundle which is where the `unexpected toke import error` occurs.

**Comment by rubenspgcavalcante** [comment]
Has the directory 'dist' been created after installation? (The plugin must be there)

**Comment by kansi** [comment]
this directory is created but the following are the content of the bundeled file

```javascript
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory();
	else if(typeof define === 'function' && define.amd)
		define([], factory);
	else if(typeof exports === 'object')
		exports["leaflet-ant-path"] = factory();
	else
		root["leaflet-ant-path"] = factory();
})(this, function() {
return /******/ (function(modules) { // webpackBootstrap
/******/ 	// The module cache
/******/ 	var installedModules = {};
/******/
/******/ 	// The require function
/******/ 	function __webpack_require__(moduleId) {
/******/
/******/ 		// Check if module is in cache
/******/ 		if(installedModules[moduleId])
/******/ 			return installedModules[moduleId].exports;
/******/
/******/ 		// Create a new module (and put it into the cache)
/******/ 		var module = installedModules[moduleId] = {
/******/ 			exports: {},
/******/ 			id: moduleId,
/******/ 			loaded: false
/******/ 		};
/******/
/******/ 		// Execute the module function
/******/ 		modules[moduleId].call(module.exports, module, module.exports, __webpack_require__);
/******/
/******/ 		// Flag the module as loaded
/******/ 		module.loaded = true;
/******/
/******/ 		// Return the exports of the module
/******/ 		return module.exports;
/******/ 	}
/******/
/******/
/******/ 	// expose the modules object (__webpack_modules__)
/******/ 	__webpack_require__.m = modules;
/******/
/******/ 	// expose the module cache
/******/ 	__webpack_require__.c = installedModules;
/******/
/******/ 	// __webpack_public_path__
/******/ 	__webpack_require__.p = "";
/******/
/******/ 	// Load entry module and return exports
/******/ 	return __webpack_require__(0);
/******/ })
/************************************************************************/
/******/ ([
/* 0 */
/***/ function(module, exports) {

	import {Polyline, MultiPolyline, polyline, multiPolyline} from "leaflet";
	import AntPath from "./components/ant-path.component";
	import antPath from "./factories/ant-path.factory";
	import MultiAntPath  from "./components/multi-ant-path.component";
	import multiAntPath from "./factories/multi-ant-path.factory";
	
	import "../style/leaflet-ant-path.sass";
	
	Polyline.AntPath = AntPath;
	polyline.antPath = antPath;
	MultiPolyline.MultiAntPath = MultiAntPath;
	multiPolyline.multiAntPat = multiAntPath;
	
	export {AntPath, antPath, MultiAntPath, multiAntPath};

/***/ }
/******/ ])
});
;
//# sourceMappingURL=leaflet-ant-path.js.map
```
This is different from what i can see [here](https://github.com/rubenspgcavalcante/leaflet-ant-path/blob/gh-pages/dist/leaflet-ant-path.js). I tried re-compiling the plugin manually 
```bash
$ npm install
$ gulp pack (or webpack or gulp dist)
```
but i can't seem to get the minified file in dist folder.

**Comment by rubenspgcavalcante** [comment]
Ok, seens to be a problem to webpack resolving where the node_modules are in the package.
https://github.com/webpack/webpack/issues/1083

Locally the build runs all ok, but on a npm install, webpack don't finds the node_modules on the parent directory. So I've corrected, tested and made a patch version   (0.1.5).

**Comment by kansi** [comment]
HUGE thanks for this :-)
